### ISSUE TITLE
Keyboard mappings on feed example "restaurant"

### ISSUE THREAD

**Issue by OliKei** [w3c/aria-practices#3144]
https://www.w3.org/WAI/ARIA/apg/patterns/feed/examples/feed/

Recommendations for lists require to use arrow keys, home and end keys and optionally "type ahead" to efficiently navigate among list items.
Lists gain focus with a single TAB key and often also loose focus on use of the TAB key. Additionally the list may also offer group skipping. 
See keyboard best practices. https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/ 

The sample does use of the recommended keystrokes, instead the user has to press TAB many times to get from one item to the next (with an indeterminate stop where the focus is not visible, which does not conform to WCAG2.2 2.4.7 Focus visible.

**Comment by mcking65** [comment]
Hi @OliKei, since a feed article can contain interactive widgets that consume arrow keys, the feed example uses page up/down. That way, it doesn't matter where focus is inside of an article, the user can immediately jump to next/prior. 

When focus is on the article, the focus ring should be drawn around the entire article. If that is not happening, that is a bug.

**Comment by JAWS-test** [comment]
In current versions of Chrome, Edge and Firefox, the focus is clearly visible for all TAB steps

**Comment by JAWS-test** [comment]
Related: https://github.com/w3c/aria-practices/issues/2243

**Comment by OliKei** [comment]
> In current versions of Chrome, Edge and Firefox, the focus is clearly visible for all TAB steps
true, it only happened on macOS Safari, however, it is not always reproducible. Today it was also working OK on Safari. Fascinating. 

Yes, interactive lists would be a great enhancement.
